### ISSUE TITLE
a11y: Missing visual label for form fields

### ISSUE THREAD

**Issue by aalves08** [rancher/dashboard#13956]
**Setup**
- Rancher version:
- Rancher UI Extensions:
- Browser type & version:

**Describe the bug**

`a11y issue: 108`
The mentioned form fields do not have visual form labels resulting in users with cognitive impairments will not be able to understand the purpose of the form fields.

Create API Key
 - Main Content 
  - Duration input field and drop-down (available after selecting the Custom radio button)  

Search modal dialog
- Type to search for a resource… input field 

Login
 - Main Content 
  - Drop-down containing "English" and " 简体中文" languages

Cluster Appearance modal dialog
- input field associated with transparent text (which gets available after selecting Badge background color)

**To Reproduce**

Go to the above mentioned UI elements and check source code on browser or use a screen reader

**Result**
The mentioned form fields do not have visual form labels resulting in users with cognitive impairments will not be able to understand the purpose of the form fields.

**Expected Result**

The form field should have visual label associate with it **OR** a screen reader label applied to them

**Screenshots**

**Additional context**

**Comment by a-arias** [comment]
**TESTS**
Rancher : 
UI:
Browser: chrome, firefox

The mentioned form fields do not have visual form labels resulting in users with cognitive impairments will not be able to understand the purpose of the form fields. ✅ 

1 - Create API Key
-Main Content ✅ 
-Duration input field and drop-down (available after selecting the Custom radio button) ✅ 

https://github.com/user-attachments/assets/84e5c6e3-8528-40cc-9572-b208a3051606
https://github.com/user-attachments/assets/44b80e7d-3848-4631-9d82-5f1d7f920480


2 -Search modal dialog
-Type to search for a resource… input field ✅ 

https://github.com/user-attachments/assets/e0db236b-b5e5-471f-801a-ce68e2f3c442
https://github.com/user-attachments/assets/50d30533-fa25-4c0f-9cba-4517b1f30767


3-Login
-Main Content ✅
-Drop-down containing "English" and " 简体中文" languages ✅
<img width="1682" alt="Image" src="https://github.com/user-attachments/assets/14a34037-39d0-47df-a48b-fc6d8f68eae5" />
https://github.com/user-attachments/assets/f33df66b-0021-47b2-8151-88e21449078a
https://github.com/user-attachments/assets/0c2c4ad4-3198-423e-bd6f-eb631ebd6b14


4-Cluster Appearance modal dialog 
input field associated with transparent text (which gets available after selecting Badge background color)✅

https://github.com/user-attachments/assets/ba2d3eaa-2d28-4c90-84b8-b77dd93f2eef
https://github.com/user-attachments/assets/9614f641-7b27-40c6-83f3-78ca2a745ab7
